### DISCUSSION TITLE
Feature Request: Page Object API Implementation in Debug Sections(Watch Section)

### DISCUSSION THREAD

**Discussion by kumar-mohan-s** [redhat-developer/vscode-extension-tester#1573] [Ideas]
We currently have APIs for handling the following sections in the debug sidebar:

Variables (components\sidebar\tree\debug\VariableSectionItem.d.ts)
Call Stack (components\sidebar\tree\debug\CallStackItem.d.ts)
Breakpoints (components\sidebar\tree\debug\BreakpointSectionItem.d.ts)
I'm looking for a similar Page Object API to handle interactions with the sections shown in the attached screenshot:
![Screenshot 2024-10-22 194158](https://github.com/user-attachments/assets/3748cfd1-cfd3-46cb-b183-2b7fcfa18fe9)

**Request:**
Could you please consider adding Page Object API implementations for these sections and include them in upcoming releases?
Thank you in advance to the developer team for considering this feature request!

**Comment by djelinek** [comment]
Hello,  I have opened feature request as follow up on this thread.

You can follow up here - https://github.com/redhat-developer/vscode-extension-tester/issues/1575

Thank you! 🙂
